docs/src/content/plugins.ts

```diff
@@ -0,0 +1,559 @@
+    tags: ["testing", "solidity"],
+  },
+];
+
+const officialPlugins: IPlugin[] = [
+  {
+    name: "@nomiclabs/hardhat-ethers",
+    author: "Nomic Labs",
+    authorUrl: "https://twitter.com/nomiclabs",
+    description: "Injects ethers.js into the Hardhat Runtime Environment",
+    tags: ["Ethers.js", "Testing", "Tasks", "Scripts"],
+  },
+  {
+    name: "@nomiclabs/hardhat-waffle",
+    author: "Nomic Labs",
+    authorUrl: "https://twitter.com/nomiclabs",
+    description:
+      "Adds a Waffle-compatible provider to the Hardhat Runtime Environment and automatically initializes the Waffle Chai matchers",
+    tags: ["Waffle", "Testing"],
+  },
+  {
+    name: "@nomiclabs/hardhat-truffle4",
+    author: "Nomic Labs",
+    authorUrl: "https://twitter.com/nomiclabs",
+    description: "Integration with TruffleContract from Truffle 4",
+    tags: ["Truffle", "Testing"],
+  },
+  {
+    name: "@nomiclabs/hardhat-truffle5",
+    author: "Nomic Labs",
+    authorUrl: "https://twitter.com/nomiclabs",
+    description: "Integration with TruffleContract from Truffle 5",
+    tags: ["Truffle", "Testing"],
+  },
+  {
+    name: "@nomiclabs/hardhat-web3",
+    author: "Nomic Labs",
+    authorUrl: "https://twitter.com/nomiclabs",
+    description: "Injects Web3 1.x into the Hardhat Runtime Environment",
+    tags: ["Web3.js", "Testing", "Tasks", "Scripts"],
+  },
+  {
+    name: "@nomiclabs/hardhat-web3-legacy",
+    author: "Nomic Labs",
+    authorUrl: "https://twitter.com/nomiclabs",
+    description: "Injects Web3 0.20.x into the Hardhat Runtime Environment",
+    tags: ["Web3.js", "Legacy", "Testing", "Tasks", "Scripts"],
+  },
+  {
+    name: "@nomiclabs/hardhat-etherscan",
+    author: "Nomic Labs",
+    authorUrl: "https://twitter.com/nomiclabs",
+    description: "Automatically verify contracts on Etherscan",
+    tags: ["Etherscan", "Verification"],
+  },
+  {
+    name: "@nomiclabs/hardhat-ganache",
+    author: "Nomic Labs",
+    authorUrl: "https://twitter.com/nomiclabs",
+    description: "Hardhat plugin for managing Ganache",
+    tags: ["Ganache", "Testing network"],
+  },
+  {
+    name: "@nomiclabs/hardhat-solpp",
+    author: "Nomic Labs",
+    authorUrl: "https://twitter.com/nomiclabs",
+    description:
+      "Automatically run the solpp preprocessor before each compilation",
+    tags: ["Solpp", "Preprocessor"],
+  },
+  {
+    name: "@nomiclabs/hardhat-solhint",
+    author: "Nomic Labs",
+    authorUrl: "https://twitter.com/nomiclabs",
+    description: "Easily run solhint to lint your Solidity code",
+    tags: ["Solhint", "Linter"],
+  },
+  {
+    name: "@nomiclabs/hardhat-vyper",
+    author: "Nomic Labs",
+    authorUrl: "https://twitter.com/nomiclabs",
+    description: "Adds support to compile Vyper smart contracts",
+    tags: ["Vyper", "Compiler"],
+  },
+  // Don't add community plugins here. They should be placed in the other array.
+].map((p) => ({
```

Comment:
not sure mixing content and code is a good idea. why we can't keep all preparing logic in the `src/model`?

Reply:
Moved logic to src/model/plugins